### ISSUE TITLE
`builtins.PerformanceAttributes` object has no attribute `pp_accuracy`

### ISSUE THREAD

**Issue by Yuri-YuzuChaN** [MaxOhn/rosu-pp-py#6]
This can be due to the following reason:
``` python
pf = Performance(**data)
attrs = pf.calculate(self.map)
obj = {
    "StarRating": attrs.difficulty.stars,
    "CS": beatmap.cs,
    "HP": beatmap.hp,
    "AR": beatmap.ar,
    "OD": beatmap.od,
    "Aim": attrs.difficulty.aim,
    "Speed": attrs.difficulty.speed,
    "aim": attrs.pp_aim,
    "speed": attrs.pp_speed,
    "accuracy": attrs.pp_accuracy,             # <- here
    "flashlight": attrs.pp_flashlight,
    "effective_miss_count": attrs.effective_miss_count,
    "pp": attrs.pp
}
```

The name of property of `PerformanceAttributes` should be `pp_acc`, instead of `pp_accuracy`:

https://github.com/MaxOhn/rosu-pp-py/blob/f0f08fe9c9a94331840db67d5cf3b9b8cd15ea55/rosu_pp_py.pyi#L896-L902

VSCode screenshot:

![94343c8ef78357fa509b1478d869771f](https://github.com/MaxOhn/rosu-pp-py/assets/65105396/c127133d-8fd0-41f7-996c-0a296b40680a)

**Comment by MaxOhn** [comment]
Would've surprised me if I didn't mess up for those at least once 😅
Will get to fixing it soon 👍

**Comment by MaxOhn** [comment]
Fix released in v1.0.1 🎉
